discovery_child_development/notebooks/openalex/06_fine_tune_embeddings.py

```diff
@@ -18,8 +18,14 @@
 )
```

Comment:
I was having a read about the tradeoffs between epochs and batch-size/ worrying about overfitting. I would also think there should be a few more epochs! Maybe we can just decide on a number? Alternatively, we could use some form of elbow method approach and run it for a lot of different epochs and see where it plateaus?

Reply:
I reckon in the short-term we could change it to the same hyperparameters Karlis used last time (in [this code](https://github.com/nestauk/innovation_sweet_spots/blob/dev/innovation_sweet_spots/analysis/notebooks/review_labelling/utils.py)) and then as a next step do a weights and biases sweep to find the best set of hyperparameters! 😱

I've been watching [this video](https://www.youtube.com/watch?v=6wRBGNLgQFU&list=PL--TX74VfiK1__SDFzriBXw5_a6rOOYqo&index=1)

Reply:
Ooo good shout!! I'm sure there must be a way to visualise with wandb! 

EarlyStoppingCallback looks super interesting!